src/vcpkg/build.cpp

```diff
@@ -1527,13 +1529,26 @@ namespace vcpkg
             .append_raw(spec_name)
             .append_raw('\n');
         result.append(msgBuildTroubleshootingMessage2).append_raw('\n');
-        result.append_indent()
-            .append_fmt_raw("https://github.com/microsoft/vcpkg/issues/"
-                            "new?template=report-package-build-failure.md&title=[{}]+Build+error",
-                            spec_name)
-            .append_raw('\n');
-        result.append(msgBuildTroubleshootingMessage3, msg::package_name = spec_name).append_raw('\n');
-        result.append_raw(paths.get_toolver_diagnostics()).append_raw('\n');
+        if (issue_body.has_value())
+        {
+            auto path = issue_body.get()->generic_u8string();
+            result.append_indent().append_fmt_raw("https://github.com/microsoft/vcpkg/issues/"
+                                                  "new?title=[{}]+Build+error&body=Copy+issue+body+from+{}\n",
+                                                  spec_name,
+                                                  Strings::percent_encode(path));
+            result.append(msgBuildTroubleshootingMessageGH).append_raw('\n');
+            result.append_indent().append_fmt_raw(
+                "gh issue create -R microsoft/vcpkg --title '[{}] Build failue' --body-file '{}'\n", spec_name, path);
```

Comment:
This needs to go through the command formatter stuff to ensure things are escaped correctly. For example, this is using single quotes which aren't a thing on Windows.

I also don't think we should be telling the user to use `gh` unless we know that is likely to succeed; at least by doing the moral equivalent of `PathSearchPath` to see if that will work in the first place.